### PR TITLE
feat: add props for message delay (VF-000)

### PR DESCRIPTION
**Pull request by JoshTheDeveloper**
add a messageDelay prop to text steps, allowing users to customize the delay messages get returned in prototype

### CODE REVIEW

File: packages/base-types/src/version/settings.ts [hunk 1]
```diff
@@ -28,16 +28,22 @@ export interface BaseVersionSettings<Prompt = unknown> {
   repeat: RepeatType;
   session: RestartSession | ResumeSession<Prompt>;
   defaultCanvasNodeVisibility: Nullable<Utils.CanvasNodeVisibility>;
+  testToolMessageDelay: number;
+  testToolMessageDelayEnabled: boolean;
```
Comment (JoshTheDeveloper) [inline packages/base-types/src/version/settings.ts:32]: yes, we want these to be separated, based on designs 
https://app.zeplin.io/project/5cfebcae362911161ad0c2ca/screen/61db2941e7d9b62c3e3681fa

File: packages/base-types/src/models/project.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ export interface ProjectPrototype {
   trainedModel?: PrototypeModel;
   lastTrainedTime?: number;
   nlp?: ProjectPrototypeNLP;
+  delayEnabled: boolean;
+  delay: number;
```
Comment (JoshTheDeveloper) [inline packages/base-types/src/models/project.ts:31]: based on designs, we want these to be separate https://app.zeplin.io/project/5cfebcae362911161ad0c2ca/screen/61db2941e7d9b62c3e3681fa

Comment (JoshTheDeveloper) [reply packages/base-types/src/models/project.ts:31]: these props are for global settings

File: packages/base-types/src/node/text.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { BaseNode, BaseStep, BaseTraceFrame, DataID, NodeNextID, StepCanvasNodeV
 
 export interface TextData extends DataID {
   content: SlateTextValue;
+  messageDelay: number;
```
Comment (JoshTheDeveloper) [inline packages/base-types/src/node/text.ts:8]: this one is isolated to specific text steps

File: packages/base-types/src/models/project.ts (hunk 2 above)
Comment (jonahsnider) [inline packages/base-types/src/models/project.ts:31]: please include a duration in this, ex.
```suggestion
  delaySeconds: number;
```

File: packages/base-types/src/node/text.ts (hunk 3 above)
Comment (jonahsnider) [inline packages/base-types/src/node/text.ts:8]: same note about durations

**Review by jonahsnider — APPROVED** [review]

**Pull request merged**